modules/perception/inference/operators/BUILD

```diff
@@ -41,4 +41,4 @@ cc_library(
 #    ],
 #)
 
-cpplint()
+cpplint()
```

Comment:
No need to change this.

Reply:
I don't know what has been modified here

Reply:
Seems that you delete the new line at the end.

Reply:
ok, may be caused by copied code from github, I will add a new line in such file.